Document using assets in .themepacks (#547)
MicrosoftDocs/win32: desktop-src/Controls/themesfileformat-overview.md

| Desktop icon | .ico                                |
| Brand logo   | .png                                |

> If you place these asset files in the root of the .cab, you can reference most of them in your .theme file directly. For example, if you have a file called `Alert.wav` in the root of your .cab, you can use it in your sound scheme:

Reviewer: ```suggestion
If you place these asset files in the root of the .cab file, you can reference the files in your .theme file. For example, a file called `Alert.wav` in the root of your .cab file could be used in your sound scheme as follows:
```

why "most of them"? Because of wallpapers below? I would still not limit it here in this sentence because the exception follows below.

Author: I am not sure why I initially caveated this statement, but looking now I'm worried that there are yet other file types that must live in other directories.

I've revised the text to be more precise, and I've specifically added "like sounds" to indicate what _definitely does_ work 😄.

That's why I wanted the theming experts to check this in 2020.

```ini
[AppEvents\Schemes\Apps\.Default\SystemAsterisk]
DefaultValue=Alert.wav
```

> Wallpaper images behave differently. When a user activates a theme, these files are extracted to a `DesktopBackground\` folder relative to the .theme file. You should include this directory in your paths in .theme, but you should still place these files at the root of the .cab. For example, if you have a wallpaper called `BestDesktop.jpg` at the root of your .cab file, you should reference it in your .cab like this:

Reviewer: ```suggestion
Wallpaper images behave differently. When a user activates a theme, the files are extracted to a `DesktopBackground\` folder relative to the .theme file. Include this directory in your paths in .theme and place these files at the root of the .cab file. For example, reference the wallpaper `BestDesktop.jpg` at the root of your .cab file like this:
```

Author: This guidance was inaccurate; I revised.
